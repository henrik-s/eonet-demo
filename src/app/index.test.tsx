/* eslint-disable no-undef */
import React from 'react';
import {App} from './index';
import * as renderer from 'react-test-renderer';

test('App renders', () => {
    const component = renderer.create(<App />,);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
