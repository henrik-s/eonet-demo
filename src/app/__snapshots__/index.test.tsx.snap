// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div
  className="container test"
>
  <p
    className="test"
  >
    Hello world
  </p>
</div>
`;
